### PR TITLE
Add nglview - biomolecular viewer in Jupyter notebook.

### PR DESCRIPTION
**Pull request by hainm**
As discussed here: https://github.com/bioconda/bioconda-recipes/issues/1305

nglview: https://github.com/arose/nglview

* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/nglview/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package:
+  name: nglview
+  version: "0.5.dev"
+
+source:
+  git_url: https://github.com/arose/nglview
```
Comment (bgruening) [inline recipes/nglview/meta.yaml]: Can you use a fixed version like this one `https://github.com/arose/nglview/archive/v0.4.tar.gz`

Comment (hainm) [inline recipes/nglview/meta.yaml]: @bgruening: so let this PR open until nglview release v0.5. The current v0.4 has a bug that causes travis failed. https://travis-ci.org/bioconda/bioconda-recipes/jobs/122976102#L1789

Comment (bgruening) [inline recipes/nglview/meta.yaml]: You can also pin a version specific to a commit, but I'm sure Alex will release a version soon :)
ping @arose

**Comment by hainm** [comment]
hi, I just updated nglview release version 0.5. Please review again. thanks.

**Comment by hainm** [comment]
Hi, should I go ahead to merge this new recipe or I need to wait for approval from one of @bioconda/core? thanks.

**Comment by tomkinsc** [comment]
Looks OK to me! Feel free to merge. 👍

**Comment by hainm** [comment]
thank you.

**Comment by tomkinsc** [comment]
Sure. In the future, if you've followed the [readme](https://github.com/bioconda/bioconda-recipes/blob/master/README.md) and the [guidelines](https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md) and everything checks out, feel free to merge recipes yourself since you're on the bioconda team. Of course, don't hesitate to @mention the team if you're unsure of something.

**Pull request merged**